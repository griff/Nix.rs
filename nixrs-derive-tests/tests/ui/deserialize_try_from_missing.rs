use nixrs_derive::NixDeserialize;

#[derive(NixDeserialize)]
#[nix(try_from = "u64")]
pub struct Test;

fn main() {}
