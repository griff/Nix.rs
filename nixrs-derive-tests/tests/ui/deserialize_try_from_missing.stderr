error[E0277]: the trait bound `Test: std::convert::From<u64>` is not satisfied
 --> tests/ui/deserialize_try_from_missing.rs
  |
  | #[nix(try_from = "u64")]
  |                  ^^^^^ the trait `std::convert::From<u64>` is not implemented for `Test`
  |
  = note: required for `u64` to implement `Into<Test>`
  = note: required for `Test` to implement `TryFrom<u64>`
